modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -14,48 +14,73 @@ endif::[]
 |500 - Bad response from authorization server, or WSDL SOAP Fault error
 |===
 
-The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request.
+The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request. In Anypoint Platform, the client ID and client secret pair is also known as credentials.
 
-The policy validates the Client ID and, optionally, the Client Secret, of a client application created within an Anypoint Platform Organization.
+After the Client ID Enforcement policy is applied, the Mule runtime engine (Mule) caches the information locally and updates it periodically. In addition to performance improvements, caching enables the policy to work offline during connectivity outages. 
 
-IMPORTANT: This policy will not validate successfully any client application credentials generated outside of the Anypoint Platform context.
+Regardless of the state of the management plane, the credentials are enforced with the existing data. Once the connectivity is re-established, the cache resumes periodical updates.
 
-NOTE: The set of Client ID and Client Secret are also known as credentials.
+//What information is stored or retrieved. -- Eva to send
+//Latest version has encryption. So the information is encrypted.
```

Comment:
I was thinking that we might not want to go into details of what is stored, mainly for security reasons. So I wouldn't expand much more on it. 

About encryption,. I have the same doubt. but if we were to put a paragraph about it, it could be a note + reference docs. 

Note: Mule Runtime can be configured to encrypt all sensitive information related to policies, contracts and initialization data.

Reply:
I added the note. What link to add?